rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnNeverTypeRector/config/configured_rule.php

```diff
@@ -9,7 +9,6 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $parameters = $containerConfigurator->parameters();
-    $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_81);
```

Comment:
biggest up-side of this PR: we get the feature supported on php7 - therefore we drop this line

Reply:
How does the test case know what PHP version is run on?
We have a convention to test both boundaries in 2 standalone test with explicit PHP version

Reply:
> How does the test case know what PHP version is run on? How does the test reader knows?
> 
> We have a convention to test both boundaries in 2 standalone test with explicit PHP version

hmm I don't know what you mean. what should I do?

Reply:
There should be test for PHP 7.4 (lower bound) and PHP 8.0 (upper bound)